docs/Advanced-Concepts/Chip-Communication.rst

```diff
@@ -109,16 +109,13 @@ reminder, to run a software RTL simulation, run:
 
 FireSim FPGA-accelerated simulations use TSI by default as well.
 
-If you would like to build and simulate a Chipyard configuration with a DTM configured for DMI communication, then you must create a
-top-level system with the DTM (``TopWithDTM``), a test-harness to connect to the DTM (``TestHarnessWithDTM``), as well as a config to use that top-level system.
+If you would like to build and simulate a Chipyard configuration with a DTM configured for DMI communication, then you must tie-off the TSI interface, and instantiate the `SimDTM`. Note that we use `WithTiedOffSerial ++ WithSimDTM` instead of `WithTiedOffDebug ++ WithSimSerial`.
```

Comment:
Is there a reason that `WithSimDTM` and `WithTiedOffDebug` are not symmetric? (i.e. `WithSimDTM` & `WithTiedOffDTM`, or `WithSimDebug` & `WithTiedOffDebug`)

Reply:
Good point, I think renaming it to `WithTiedOffDTM` makes sense.

Reply:
Technically DTM is separate from DMI and JTAG. Do we want these mixins to state what's inside the system or what the interface is? Maybe we should name it something with DMI and JTAG like `WithSimDMI` `WithSimJTAG` if we are considering doing the latter.

Reply:
That doesn't work because `Debug.connectDebug` will connect either SimDTM or SimJTAG, depending on the interface of the debug module.
SimDTM and SimJTAG are the names of the actual widgets, so I don't think renaming this to SimDMI /SimJTAG is a good idea.

Given the behavior of `Debug.connectDebug`, I think `WithSimDTM` is the right name here.

Reply:
Sorry. All `WithSimDTM` does is connect a `Debug` module in the `TestHarness` to the JTAG/DMI signals. I think this distinction should be mentioned in the docs. I don't think it is immediately clear that adding `WithSimDTM` connects both JTAG or DMI.

Reply:
Ok, perhaps I'll name it `WithSimDebug` and `WithTiedOffDebug` then. That seems to be more accurate.

Reply:
I agree with `WithSimDebug`.